src/components/FullscreenModal/FullscreenModal.enums.ts

```diff
@@ -0,0 +1,12 @@
+export const FullscreenModalSizes = {
+  lg: 'lg',
+  md: 'md',
+  sm: 'sm',
+} as const;
+
+export const FullscreenModalLayouts = {
+  single6: 'single-6',
+  single8: 'single-8',
+  sidebar46: 'sidebar-4-6',
```

Comment:
I see only one layout in the design specs with sidebar. Are the designs not complete and this is a direct request from @MateCziner ? Also the 10 columns layout is not in sync with the 12 columns global Grid. Can you please tell more about this decision?

Reply:
@youssefssc these layout was requested by @MateCziner and are not implemented in Figma. Not sure what you mean by: `Also the 10 columns layout is not in sync with the 12 columns global Grid`

Reply:
Nevermind the second comment. I thought the 4-6 is a new 10 columns layout but found it's actually 12 cols and the sidebar is just offseted by 1 col